package/MDAnalysis/analysis/rms.py

```diff
@@ -533,33 +539,8 @@ def save(self, filename=None):
         return filename
 
 
-class RMSF(object):
-    """Class to perform RMSF analysis on a set of atoms across a trajectory.
-
-    Run the analysis with :meth:`RMSF.run`, which stores the results
-    in the array :attr:`RMSF.rmsf`.
-
-    This class performs no coordinate transforms; RMSFs are obtained from atom
-    coordinates as-is.
-
-    .. versionadded:: 0.11.0
-    """
-
-    def __init__(self, atomgroup):
-        """Calculate RMSF of given atoms across a trajectory.
-
-        Parameters
-        ----------
-        atomgroup : mda.AtomGroup
-                AtomGroup to obtain RMSF for
-
-        """
-        self.atomgroup = atomgroup
-        self._rmsf = None
-
-    def run(self, start=None, stop=None, step=None, progout=10,
-            verbose=None, quiet=None):
-        """Calculate RMSF of given atoms across a trajectory.
+class RMSF(AnalysisBase):
+    """Calculate RMSF of given atoms across a trajectory.
```

Comment:
Please leave the original versionadded markup and add a new versionchanged ("Follows analysis API").